webapp/src/cljc/lipas/roles.cljc

```diff
@@ -0,0 +1,94 @@
+(ns lipas.roles)
+
+(def privileges
+  {:create {:doc "Oikeus lisätä liikuntapaikkoja"}
+   :view {:doc "Oikeus nähdä liikuntapaikat ja niihin liittyvät perustiedot
+               ja lisätiedot"}
+   :edit {:doc "Oikeus muokata liikuntapaikkoja"}
+
+   :view-activity {:doc "Nähdä UTP tiedot"}
+   :edit-activity {:doc "Oikeus muokata UTP tietoja"}
+
+   :create-loi {:doc "Oikeus luoda Muita kohteita"}
+   :view-loi {:doc "Oikeus nähdä Muita kohteita"}
+   :edit-loi {:doc "Oikeus muokata Muita kohteita"}
+
+   :view-floorball {:doc "Oikeus nähdä Salibandy lisätiedot"}
+   :edit-floorball {:doc "Oikeus muokata Salibandy lisätietoja"}
+
+   :analysis-tool {:doc "Oikeus käyttää analyysityökalua"}
+
+   :user-self {:doc "Oikeus omiin käyttäjätietoihin (salasanan vaihto jne)"}
+   :user-management {:doc "Käyttäjien hallinta (admin)"}})
```

Comment:
In the future we _might_ need to add also organization level admins, that can manage organization's users. I'm not sure if we really really need that though. But I guess the new model would adapt to also that. E.g. role :org-admin can have :user-management restricted to users with org-id 123 or something. 🤔

Reply:
Yes, org-id would be additional role-context value.